src/Atmos/Model/atmos_tendencies.jl

```diff
@@ -36,3 +44,29 @@ eq_tends(pv::PV, ::AtmosModel, ::Flux{FirstOrder}) where {PV <: Energy} =
 # TotalMoisture
 eq_tends(pv::PV, ::AtmosModel, ::Flux{FirstOrder}) where {PV <: TotalMoisture} =
     ()
+
+#####
+##### Second order fluxes
+#####
+
+# Mass
+moist_diffusion(pv::PV, ::DryModel) where {PV <: Mass} = ()
+moist_diffusion(pv::PV, ::MoistureModel) where {PV <: Mass} =
+    (MoistureDiffusion{PV}(),)
```

Comment:
This name was not introduced by this PR, but just a warning that a line like "MoistureDiffusion{PV}" is primed to confuse Earth scientists, because the term "PV" is often used as an abbreviation for ["potential vorticity"](https://en.wikipedia.org/wiki/Potential_vorticity). Most of the places where "PV" shows up there is enough context to infer that it refers to something other than potential vorticity. But in lines like these (and lines like `Advect{PV}` it's not quite so clear. I certainly did a double-take, myself, even though I know julia...

Reply:
Do you, or anyone else (cc @yairchn, @trontrytel) have a better name for this? What's wrong with `Advect`? I'm happy to change it/them. Regarding `PV`, it's the most sensible short type name for `PrognosticVariable` (the abstract type of `Mass`, `TotalMoisture` etc.) I think, but again, I'm open to changing. Perhaps changing `PV` and `Advect` would be better for a separate PR, as that would need to happen across the board--and it would be nice to see how everything changes at once.

Reply:
It's just PV that conflicts with other abbreviations "PV". I wouldn't change it now, just something to keep in the back of your mind so if you see an opportunity to change it, you can.

Reply:
I agree. I also think of vorticity first when I see PV. But I don't have a good 2-letter replacement for this. So I got used to it.

Reply:
Use `PVAR`, there's no reason it has to be 2 letters.

Reply:
Maybe we can change to `PVAR` across the board in a separate PR?